yolov3/README.md

```diff
@@ -0,0 +1,104 @@
+# YOLOv3
+
+## Description
+This model is a real-time neural network for object detection that detects 80 different classes.
+
+## Model
+
+|Model        |Download  |Checksum|ONNX version|Opset version|Accuracy |
+|-------------|:--------------|:--------------|:--------------|:--------------|:--------------|
+|YOLOv3       |[237 MB](https://onnxzoo.blob.core.windows.net/models/opset_10/yolov3/yolov3.onnx) | [MD5](https://onnxzoo.blob.core.windows.net/models/opset_10/yolov3/yolov3-md5.txt) |1.4.1 |10 |mAP of 0.553 |
+
+
+
+<hr>
+
+## Inference
+
+### Input to model
+Image shape `(1x3x416x416)`
+
+### Preprocessing steps
+The images have to be loaded in to a range of [0, 1]. The transformation should preferrably happen at preprocessing.
+
+The following code shows how to preprocess a NCHW tensor:
+
+```python
+import numpy as np
+from PIL import Image
+
+# this function is from yolo3.utils.letterbox_image
+def letterbox_image(image, size):
+    '''resize image with unchanged aspect ratio using padding'''
+    iw, ih = image.size
+    w, h = size
+    scale = min(w/iw, h/ih)
+    nw = int(iw*scale)
+    nh = int(ih*scale)
+
+    image = image.resize((nw,nh), Image.BICUBIC)
+    new_image = Image.new('RGB', size, (128,128,128))
+    new_image.paste(image, ((w-nw)//2, (h-nh)//2))
+    return new_image
+
+# img = Image.open(img_path)
+def preprocess(img):
+    model_image_size = (416, 416)
+    boxed_image = letterbox_image(img, tuple(reversed(model_image_size)))
+    image_data = np.array(boxed_image, dtype='float32')
+    image_data /= 255.
+    image_data = np.transpose(image_data, [2, 0, 1])
+    image_data = np.expand_dims(image_data, 0)
+    return image_data
+```
+
+### Output of model
+The model has 3 outputs.
+boxes: `(1x'n_candidates'x4)`, the coordinates of all anchor boxes,
+scores: `(1x80x'n_candidates')`, the scores of all anchor boxes per class,
```

Comment:
where is the list of classes?

Reply:
The 3rd output is the indices from the boxes tensor, I.e., (batch_index, class_index, box_index). So it contains class_index. The list of classes is coco_classes. we can find it here:
https://github.com/qqwweee/keras-yolo3/blob/master/model_data/coco_classes.txt
will add a link here.